templates/emails/transactional/test-email.txt

```diff
@@ -5,7 +5,7 @@ Hi {{name}}
 
 This is a test to make sure that you receive emails sent to your alias {{alias}}.
 
-If you have any questions, feel free to reply to this email.
+If you have any questions, feel free to contact us through our support page: {{support_page}}
```

Comment:
We can remove this sentence as the support is also mentioned in the text email

<img width="874" alt="image" src="https://user-images.githubusercontent.com/3167780/168320270-ebfac571-447c-4dad-8296-107df291243a.png">

Reply:
Removed it too